### PR TITLE
8538-One-Rate-Multiple-Types-Shipper-Options => master

### PR DESCRIPTION
**Pull request by dustinmoorenet**
### Added new amamzon fedex one rate options

ordoro/ordoro#5838

ordoro/shipper-options@839881e3c343c6efbb8e11f515ca58b76821bf8c

### CODE REVIEW

**Comment by dustinmoorenet** [comment]
I am going to blame that typo on the lunch refreshments

File: ShipperOptions.json [hunk 1]
```diff
@@ -1964,6 +1964,26 @@
...
+      {
+        "value": "FEDEX_PTP_PRIORITY_OVERNIGHT_ONE_RATE",
+        "display": "FedEx® Priority Overnight® One Rate®"
+      },
+      {
+        "value": "FEDEX_PTP_STANDARD_OVERNIGHT_ONE_RATE",
+        "display": "FedEx® Standard Overnight® One Rate®"
+      },
+      {
+        "value": "FEDEX_PTP_SECOND_DAY_AM_ONE_RATE",
```
Comment (sophieher) [inline ShipperOptions.json]: if we're looking at typos, does there need to be a space before `A.M.`?

File: ShipperOptions.json [hunk 2]
```diff
@@ -1964,6 +1964,26 @@
...
+        "display": "FedEx® 2Day®A.M. One Rate®"
+      },
+      {
+        "value": "FEDEX_PTP_SECOND_DAY_ONE_RATE",
+        "display": "FedEx® 2Day® One Rate®"
+      },
+      {
+        "value": "FEDEX_PTP_EXPRESS_SAVER_ONE_RATE",
+        "display": "FedEx® Express Saver®"
```
Comment (sophieher) [inline ShipperOptions.json]: Should the display value also say `One Rate`?

File: ShipperOptions.json (hunk 1 above)
Comment (dustinmoorenet) [reply ShipperOptions.json]: sure

File: ShipperOptions.json (hunk 2 above)
Comment (dustinmoorenet) [reply ShipperOptions.json]: also, blame the beer.

**Pull request merged**